src/bitcoinForks/index.ts

```diff
@@ -50,14 +50,16 @@ export default class BitcoinWallet implements Partial<IWallet> {
   segwitInternal: string;
   network: any;
   addressDB: AddressDB | undefined;
+  transactionDB: TransactionDB | undefined;
   walletId: string;
 
   constructor(
     xpub: string,
     coinType: string,
     walletId: string,
     zpub?: string,
-    addressDb?: AddressDB
+    addressDb?: AddressDB,
+    transactionDb?: TransactionDB
   ) {
```

Comment:
The params are getting large, use object parameters here `{ xpub: string .... })`